tests/conftest.py

```diff
@@ -188,3 +189,14 @@ def test_timeout_ctrl_time(monkeypatch):
 def tmpfile(tmpdir):
     tmpfile = f'jina_test_{next(tempfile._get_candidate_names())}.db'
     return tmpdir / tmpfile
+
+
+@pytest.yield_fixture(scope="session")
+def event_loop(request):
```

Comment:
Cant we use the default loop fixture? If so (I think the scope is different), the comment below seems to be wrong as the scope here session and not 'for each test'

Reply:
You're right. The comment is wrong. It needs to be a session, else we need to handle event loop (start and closure) logic in aiohttp client session, which is not required in the actual code (as uvicorn takes care of it), and needed only in tests. 

I had to spent >2hrs just to fix this :)